src/chapters/loops/studio.rst

```diff
@@ -40,51 +43,65 @@ Part A (Put dinner together)
 
         ['apple', 'banana', 'more kale', 'ice cream', 'chocolate', 'kiwi']
 
-`Code it at replt.it <https://repl.it/@launchcode/LoopstudiopartsAandC>`__
 
-2. Use a ``for`` loop to assemble 6 meals.
+2. Inside of ``mealAssembly()``, write a ``for`` loop to assemble ``numMeals`` meals.
 
-   a. The meals must include one item from each of the source arrays.
+   a. The meals must include one item from each category in the ``pantry`` array.
```

Comment:
i think a hint about nesting loops might help here.  i remember as a student not knowing that you could do this for the studio.  it made for a very rough studio.

maybe something like this? 
The computer needs to know how many crew members to prepare food for and what ingredients.  Consider creating a nested loop that will create a meal for each crew member and then add it into a larger collection of meals.

Reply:
thanks!